sections/disciplines.md

```diff
@@ -0,0 +1,45 @@
+Disciplines
+=========
+
+Disciplines currently must be created and assigned through the UI.
+
+Endpoints:
+
+- [Get Disciplines](#get-disciplines)
+- [Get a Discipline](#get-a-discipline)
+
+
+Get Disciplines
+-------------
+
+* `GET /api/v1/discipline` will return a [paginated list](/README.md#pagination) of Disciplines.
+
+###### Example JSON Response
+
+```json
+[
+  {
```

Comment:
Yea, I left this out as I felt it should be documented in one place and linked to. It felt like clutter in a sense to have it in this example too, but I can see the value and can add it.